tests/examples/tokens/test_erc1155.py

```diff
@@ -0,0 +1,316 @@
+
+    # burn existing
+    erc1155.burnBatch([11,12],[1,1],transact={"from": w3.eth.accounts[3]})
+
+    assert (lambda: erc1155.balanceOfBatch([w3.eth.accounts[1],w3.eth.accounts[1],w3.eth.accounts[1]],[11,12,13], transact={"from": w3.eth.accounts[1]}) == [0,0,1])
+
+    # burn again, should revert
+    assert_tx_failed(lambda: erc1155.burnBatch([1,2],[1,1],transact={"from": w3.eth.accounts[1]}))
+
+    assert (lambda: erc1155.balanceOfBatch([w3.eth.accounts[1],w3.eth.accounts[1],w3.eth.accounts[1]],[1,2,3], transact={"from": w3.eth.accounts[1]}) == [0,0,1])
+
+
+def test_approval_functions(erc1155, w3, test_mint_batch, assert_tx_failed):
+
+    # self-approval by the owner
+    assert_tx_failed(lambda: erc1155.setApprovalForAll(w3.eth.accounts[5],w3.eth.accounts[5],True, transact={"from": w3.eth.accounts[5]}))
+
+    # let's approve and operator for somebody else's account
+    assert_tx_failed(lambda: erc1155.setApprovalForAll(w3.eth.accounts[0],w3.eth.accounts[5],True, transact={"from": w3.eth.accounts[3]}))
+
+    # set approval correctly
+    erc1155.setApprovalForAll(w3.eth.accounts[0],w3.eth.accounts[5],True)
+
+    # check approval
+    erc1155.isApprovedForAll(w3.eth.accounts[0],w3.eth.accounts[5])
+
+    # remove approval
+    erc1155.setApprovalForAll(w3.eth.accounts[0],w3.eth.accounts[5],False)
+
+
+def test_max_batch_size_violation(erc1155, w3, assert_tx_failed):
+    TOTAL_BAD_BATCH = 200
+    ids = []
+    amounts = []
+    for i in range(1,TOTAL_BAD_BATCH):
+        ids.append(i)
+        amounts.append(1)
+
+    assert_tx_failed(lambda: erc1155.mintBatch(w3.eth.accounts[1], ids, amounts, '', transact={"from": w3.eth.accounts[0]}))
+
+
+# Transferring back and forth
+
+def test_ownership_functions(erc1155, w3, assert_tx_failed,tester):
+    owner, a1, a2 = w3.eth.accounts[0:3]
+    print(owner,a1,a2)
+    print('___owner___', erc1155.owner())
+    # change owner from account 0 to account 1 and back
+    
+    assert erc1155.owner() == owner
+    assert_tx_failed(lambda: erc1155.transferOwnership(w3.eth.accounts[1], transact={"from": a2}))
+    
+    # try to transfer ownership to current owner
+    assert_tx_failed(lambda: erc1155.transferOwnership(w3.eth.accounts[0]))
+    # try to transfer ownership to ZERO ADDRESS
+    assert_tx_failed(lambda: erc1155.transferOwnership("0x0000000000000000000000000000000000000000"))
+
+    # Transfer ownership to account 1    
+    erc1155.transferOwnership(w3.eth.accounts[1])
+    assert (lambda: erc1155.owner() == w3.eth.accounts[1])
+
+def test_renounce_ownership(erc1155, w3, assert_tx_failed):
+    
+    assert erc1155.owner() == w3.eth.accounts[0]
+    # try to transfer ownership from non-owner account
+    assert_tx_failed(lambda: erc1155.renounceOwnership(transact={"from": w3.eth.accounts[2]}))
+
+    erc1155.renounceOwnership(transact={"from": w3.eth.accounts[0]})
+
+    assert (lambda: erc1155.owner() == ZERO_ADDRESS)
```

Comment:
```suggestion
    assert erc1155.owner() == ZERO_ADDRESS
```

Reply:
Not sure when you are doing the lambdas

Reply:
I'm doing the lambdas because without it returns 'None'. That's why I had None there in the first place.

Most assertions I use the lambda approach because it returns transaction IDs instead of the actual return value.

Reply:
This is very strange and unexpected behavior

Reply:
With brownie, it returns ZERO_ADDRESS and pretty much the same test with eth-tester returns 'None'.

What would you like to do in this case?
For now I put the line `    assert erc1155.owner() == None` back to check the owner.